governance/executive-council.md

```diff
@@ -0,0 +1,267 @@
+
+Lazy consensus is applied for decisions such as those listed above that do not require a formal vote.
+
+Lazy consensus means that decisions are approved by default unless a member of
+the Executive Council explicitly objects within a predefined period.
+This allows decisions to move forward efficiently while providing an
+opportunity for discussion.
+
+When objections are raised, further discussion occurs. This ensures decisions reflect the collective agreement of the Council
+while minimizing unnecessary delays.
+
+### How the Executive Council meets and communicates
+
+#### Monthly meetings
+
+The Executive Council holds **monthly meetings** (typically 1.5 hours) to discuss key topics, review updates, and make decisions by consensus. During these meetings, the Executive Director (ED) provides updates on the organization’s development, growth, and operations and may request feedback on specific issues. This time is also used for strategic planning and discussion.
+
+During these meetings, Council members take notes. These notes are summarized and published in the [Executive Council Minutes section of this Handbook](https://github.com/pyOpenSci/handbook/tree/main/reference/meeting-notes/executive-council).  Once the notes are merged, they are [publicly available in this Handbook](https://www.pyopensci.org/handbook/reference/meeting-notes/executive-council/intro.html).
+
+
+**Ad hoc Executive Council meetings** are scheduled as needed to address pressing issues.
+
+**Executive Director--Executive Council Chair meetings:** The Executive Director also meets regularly with the pyOpenSci Board Chair to ensure alignment on key issues and strategic direction.
+
+
+### Communication platforms & tools
+
+#### Slack: primary communication platform
+
+The **[Executive Council Slack channel](pyos-slack)** is the primary tool for communication between monthly meetings. All updates, discussions, and requests for input happen here; email is rarely used for EC communications.
+EC members are expected to check Slack semi-regularly to stay informed.
+
+- **Feedback requests:** It is understood that EC members have jobs and lives outside of their EC responsibilities. As such, when the Executive Director (ED) requests input on Slack, they will specify what feedback is needed and provide clear response timelines and areas to focus on.
+- **Transparency:** The Slack channel is also used to share updates about changes that occur outside of formal monthly meetings. Examples of such updates include new hires, event updates, and celebrating successes.
+
+#### Google Drive: document sharing and collaboration
+
+The pyOpenSci Executive Council uses a shared **[Google Drive](google-drive)** within the pyOpenSci Google Workspace to store and share documents. This includes meeting notes, agendas, and other important materials. Members are encouraged to access and update shared resources using the shared Drive.
+
+### Selection of Executive Council members
```

Comment:
This is a part of the governance structure, so we can move this section up to the top section under 'Decision-making process'